src/eei.h

```diff
@@ -47,6 +50,35 @@ class WasmEngine {
   ) = 0;
 
   virtual void verifyContract(std::vector<uint8_t> const& code) = 0;
+
+  static void enableBenchmarking() noexcept { benchmarkingEnabled = true; }
+
+protected:
+  void instantiationStarted() noexcept
+  {
+    if (benchmarkingEnabled)
+      instantiationStartTime = clock::now();
+  }
+
+  void executionStarted() noexcept
+  {
+    if (benchmarkingEnabled)
+      executionStartTime = clock::now();
+  }
+
+  void executionFinished()
+  {
+    if (benchmarkingEnabled)
+      collectBenchmarkingData();
+  }
+
+private:
+  void collectBenchmarkingData();
+
+  using clock = std::chrono::high_resolution_clock;
+  static bool benchmarkingEnabled;
```

Comment:
I'm suggesting to set this here to `false`.

Reply:
Static member variables must be initialized outside of class.

Reply:
```
/home/chfast/Projects/ewasm/hera/src/eei.h:79:15: error: ISO C++ forbids in-class initialization of non-const static member ‘hera::WasmEngine::benchmarkingEnabled’
   static bool benchmarkingEnabled = false;
               ^~~~~~~~~~~~~~~~~~~
```